app/models/likes.rb

```diff
@@ -0,0 +1,10 @@
+class Like < ApplicationRecord
+  belongs_to :author
```

Comment:
- [ ] The same happens here:

```suggestion
  belongs_to :author, class_name: 'User'
```

Reply:
Thank you for your kind feedback, working on it @anagudelogu.